### PR TITLE
IGNITE-22702 Sql. Rewrite CREATE TABLE FROM QUERY RESULT in tests

### PR DESCRIPTION
**Pull request by xtern**
https://issues.apache.org/jira/browse/IGNITE-22702

1. `CREATE TABLE ... AS SELECT` is rewritten in two statements.
2.  Some parts of tests muted with another issues (`test_filter_clause.test_ignore`  and `test_order_variable_size_payload.test_ignore`)

### CODE REVIEW

File: modules/sql-engine/src/integrationTest/sql/aggregate/aggregates/test_group_by_many_groups.test [hunk 1]
```diff
@@ -1,11 +1,15 @@
-# name: test/sql/aggregate/aggregates/test_group_by_many_groups.test_slow
+# name: test/sql/aggregate/aggregates/test_group_by_many_groups.test
```
Comment (xtern) [inline modules/sql-engine/src/integrationTest/sql/aggregate/aggregates/test_group_by_many_groups.test:1]: Test executes about 8 seconds on teamcity:
https://ci.ignite.apache.org/project.html?projectId=ApacheIgnite3xGradle_Test_IntegrationTests&testNameId=3419856454515980994&tab=testDetails

File: modules/sql-engine/src/integrationTest/sql/order/test_order_variable_size_payload.test_ignore [hunk 2]
```diff
@@ -206,23 +217,7 @@ statement ok
...
-(5, '0reallylongstring1', 4, '1reallylongstring8'),
-(5, '0reallylongstring1', 4, '1reallylongstring7'),
-(5, '0reallylongstring2', 3, '1reallylongstring8'),
-(5, '0reallylongstring2', 3, '1reallylongstring7'),
-(5, '0reallylongstring2', 4, '1reallylongstring8'),
-(5, '0reallylongstring2', 4, '1reallylongstring7')
+INSERT INTO test6 VALUES (6, '0reallylongstring1', 3, '1reallylongstring8'), (6, '0reallylongstring1', 3, '1reallylongstring7'),(6, '0reallylongstring1', 4, '1reallylongstring8'),(6, '0reallylongstring1', 4, '1reallylongstring7'),(6, '0reallylongstring2', 3, '1reallylongstring8'),(6, '0reallylongstring2', 3, '1reallylongstring7'),(6, '0reallylongstring2', 4, '1reallylongstring8'),(6, '0reallylongstring2', 4, '1reallylongstring7'),(5, '0reallylongstring1', 3, '1reallylongstring8'),(5, '0reallylongstring1', 3, '1reallylongstring7'),(5, '0reallylongstring1', 4, '1reallylongstring8'),(5, '0reallylongstring1', 4, '1reallylongstring7'),(5, '0reallylongstring2', 3, '1reallylongstring8'),(5, '0reallylongstring2', 3, '1reallylongstring7'),(5, '0reallylongstring2', 4, '1reallylongstring8'),(5, '0reallylongstring2', 4, '1reallylongstring7')
```
Comment (ygerzhedovich) [inline modules/sql-engine/src/integrationTest/sql/order/test_order_variable_size_payload.test_ignore:220]: What reason for the change? We don't support multilines in SQL logic tests?

File: modules/sql-engine/src/integrationTest/sql/order/test_order_variable_size_payload.test_ignore [hunk 3]
```diff
@@ -397,10 +392,10 @@ NULL	[[d, d, d]]
 
 # lists longer than STANDARD_VECTOR_SIZE
 statement ok
-CREATE TABLE test9 AS (SELECT range % 4 i, range j FROM range(5000));
+CREATE TABLE test9(i INT, j INT)
 
 statement ok
-INSERT INTO test9 VALUES (0, NULL), (1, NULL), (2, NULL), (3, NULL)
```
Comment (ygerzhedovich) [inline modules/sql-engine/src/integrationTest/sql/order/test_order_variable_size_payload.test_ignore:403]: why did you remove the insert?

File: modules/sql-engine/src/integrationTest/sql/order/test_order_variable_size_payload.test_ignore (hunk 2 above)
Comment (xtern) [reply modules/sql-engine/src/integrationTest/sql/order/test_order_variable_size_payload.test_ignore:220]: Seems so, for me test fails with the following
```
org.opentest4j.AssertionFailedError: Not expected result at: (test_order_variable_size_payload.test:27). Statement: INSERT INTO test0 VALUES ('Shipping and Receiving Supervisor', 'Ackerman'),

	at org.apache.ignite.internal.sql.sqllogic.Statement.execute(Statement.java:112)
	at org.apache.ignite.internal.sql.sqllogic.SqlScriptRunner.run(SqlScriptRunner.java:70)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:834)
Caused by: org.apache.ignite.sql.SqlException: IGN-SQL-3 TraceId:ba640ebb-929e-43c5-84da-fbd9d2b39cd4 Failed to parse query: Encountered "," at line 1, column 75
```

Comment (xtern) [reply modules/sql-engine/src/integrationTest/sql/order/test_order_variable_size_payload.test_ignore:220]: looks like this, for me test fails with the following
```
org.opentest4j.AssertionFailedError: Not expected result at: (test_order_variable_size_payload.test:144). Statement: INSERT INTO test6 VALUES (6, '0reallylongstring1', 3, '1reallylongstring8'),

	at org.apache.ignite.internal.sql.sqllogic.Statement.execute(Statement.java:112)
	at org.apache.ignite.internal.sql.sqllogic.SqlScriptRunner.run(SqlScriptRunner.java:70)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:834)
Caused by: org.apache.ignite.sql.SqlException: IGN-SQL-3 TraceId:eef5308c-7505-416c-b779-31ca06190e87 Failed to parse query: Encountered "," at line 1, column 76
```

File: modules/sql-engine/src/integrationTest/sql/order/test_order_variable_size_payload.test_ignore (hunk 3 above)
Comment (xtern) [reply modules/sql-engine/src/integrationTest/sql/order/test_order_variable_size_payload.test_ignore:403]: Removed by mistake , fixed, thanks.

**Review by ygerzhedovich — APPROVED** [review]
